### DISCUSSION TITLE
win7双击无法运行

### DISCUSSION THREAD

**Discussion by vinsoft123** [getlantern/lantern-forum-cn#193] [General]
win7双击无法运行，我的票号是69752135，期待解决问题

**Comment by MaxMKMax** [comment]
@vinsoft123 请问是哪个地方或页面双击没反应呢?

**Reply by vinsoft123** [reply]
win7双击蓝灯客户端，无法运行

**Reply by MaxMKMax** [reply]
@vinsoft123 好的谢谢您. 我们尽快解决问题.

**Comment by vinsoft123** [comment]
你好，

我们已收到您关于无法登录 Lantern 帐户的反馈。 目前，Lantern 不需要任何登录方法即可使用我们的免费版本，一旦将应用程序下载到您的首选设备上，您只需打开/关闭 Lantern 即可随意使用。 如果您的问题涉及登录付费 Pro 帐户，请参阅本文，其中概述了恢复和链接 Lantern Pro 帐户。

真挚地，
The Lantern Team
～～～～～～～～～～～～～～～
以上是lantern team的回信，可以说是毫无用处。
在windows系统中，双击 lantern运行程序后，没有反应，即右下角没有lantern的图标，更别提更改设置，添加设备了。。。您的回复完全没有用噢。并且我下载并重新安装了lantern的新版本，还是无法正常使用。by the way ,我是付费用户。

**Comment by MaxMKMax** [comment]
@vinsoft123 您好! 想跟您分享一个好消息! 很高兴告诉您我们已经解决了Window7的问题。这是您可以下载
蓝灯Window 7 的链接。
https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-win7.exe
